### PR TITLE
added r_cons_break_{push/pop} to handle ^C better

### PR DESCRIPTION
**Pull request by alvarofe**
Besides an UAF has been fixed afecting only ELF

### CODE REVIEW

**Comment by alvarofe** [comment]
my last commit breaks some test in a2f but is because i fixed warnings when rebasing.

For example 

```
+void addTarget(RCore *core, RStack *stack, Sdb *db, ut64 addr) {
+       if (!sdb_num_get (db, Fhandled(addr), NULL)) {
+               r_stack_push (stack, addr);
+               sdb_num_set (db, Fhandled(addr), 1, 0);
+       }
+}
```

r_stack_push's second argument expects a pointer no constant value, `r_stack_push (stack, &addr)` should be used.

The same happens with 

```
        while (!r_stack_is_empty (stack)) {
                block_end = false;
                addr = r_stack_pop (stack);
``` 

it should be `addr = *(ut64*)r_stack_pop (stack)` I can revert those changes and @rlaemmert can take a look with more calm.

**Comment by alvarofe** [comment]
Plus this change
```
 static ut64 getFunctionSize(Sdb *db) {
-       ut64 min, max;
+       ut64 min = UT64_MAX, max = 0;
```

File: libr/cons/dietline.c [hunk 1]
```diff
@@ -464,13 +507,14 @@ R_API const char *r_line_readline_cb_win(RLineReadCallback cb, void *user) {
 	I.buffer.data[0] = '\0';
 	if (I.contents) {
 		memmove (I.buffer.data, I.contents,
-			R_MIN (strlen (I.contents)+1, R_LINE_BUFSIZE-1));
+			 R_MIN (strlen (I.contents) + 1, R_LINE_BUFSIZE - 1));
```
Comment (radare) [inline libr/cons/dietline.c]: w

Comment (radare) [inline libr/cons/dietline.c]: whitespace

**Comment by radare** [comment]
conflict with dwarf.c

**Comment by alvarofe** [comment]
merged

**Pull request closed without merging**